webpack.config.js

```diff
@@ -200,8 +192,8 @@ module.exports = [
                 publicPath: `${STATIC_PATH}/`
             },
             externals: {
-                React: 'react',
-                ReactDOM: 'react-dom'
+                'react': 'react',
+                'react-dom': 'react-dom'
```

Comment:
This PR caused me to read up on how webpack handles `externals`. If I understand correctly, `externals: {a: 'b'}` means "when I try to import `a`, assume it's already available in a global variable called `b`" -- so this line is saying "when I try to import `react-dom`, assume that it's already available in a global variable called `react-dom`."

Is that really the variable name? I guess it must be since your tests were successful. It just seems odd that they'd use a name with a dash / minus in it. I suppose webpack uses something like `window['react-dom']` to access it (or maybe `self` or `global` instead of `window`).

Reply:
Good question! 

So you are right if we were building directly for the browser, this syntax would connect require(key) to window[value] (in terms of keys and values of the object). But we are building a UMD module, so it will end up mapping require(key) to require(value). With umd as the output libraryTarget you can actually specify multiple mappings depending on how the module is being used [like this](https://webpack.js.org/configuration/externals/#object) which might make it easier to hack on it with script tags, but didn't seem like an immediate need.